packages/layers/auth/app/composables/useConnectTosModals.ts

```diff
@@ -0,0 +1,46 @@
+export const useConnectTosModals = () => {
+  const { baseModal } = useConnectModal()
+  const { logout } = useConnectAuth()
+  const t = useNuxtApp().$i18n.t
+
+  function openDeclineTosModal() {
+    baseModal.open({
+      title: `${t('connect.label.declineTermsOfUse')}?`,
+      description: t('connect.text.declineTOSCantAccessService'),
+      dismissible: true,
+      buttons: [
+        {
+          label: t('connect.label.declineTermsOfUse'),
+          onClick: async () => await logout() // TODO: - QUESTION - configure redirect target on logout?
```

Comment:
I don't think so. This is would be such a rare flow that I don't think any app will want their own flow for it, but we can easily expose this later if there's a need